### ISSUE TITLE
Annotations are not detected in functions

### ISSUE THREAD

**Issue by marcospassos** [pepakriz/phpstan-exception-rules#45]
```php
/**
  * @throws \OutOfBoundsException
  */
function foo()  {
    throw new \OutOfBoundsException();
}
```

Warning:
```
Missing @throws OutOfBoundsException annotation  
```

**Comment by pepakriz** [comment]
It has been fixed in the latest release.
Thank you!

**Comment by marcospassos** [comment]
Thank you very much for spending time fixing it.
